catalystwan/utils/config_migration/converters/feature_template/base.py

```diff
@@ -1,8 +1,32 @@
-from typing_extensions import Protocol
+# Copyright 2024 Cisco Systems, Inc. and its affiliates
+from typing import Optional, Tuple
 
+from catalystwan.models.configuration.config_migration import ConvertResult
 from catalystwan.models.configuration.feature_profile.parcel import AnyParcel
+from catalystwan.utils.config_migration.converters.exceptions import CatalystwanConverterCantConvertException
 
 
-class FeatureTemplateConverter(Protocol):
-    def create_parcel(self, name: str, description: str, template_values: dict) -> AnyParcel:
-        ...
+class FTConverter:
+    """Feature Template Converter base class."""
+
+    supported_template_types: Tuple = tuple()
+
+    def __init__(self) -> None:
+        if not self.supported_template_types:
```

Comment:
what do you think about returning the "unsupported" conversion_result instead of raising the exception?

Reply:
Sure good idea :D